qiskit/transpiler/passes/a_star_mapper/setup.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+"""The A-star Mapper
+"""
+
+# import statements
+# import os
+import sys
+import distutils.sysconfig
+from setuptools import setup, Extension
+from Cython.Build import cythonize
+from Cython.Distutils import build_ext
+
+
+DOCLINES = __doc__.split("\n")
+
+CLASSIFIERS = """\
+Development Status :: 4 - Beta
+Intended Audience :: Science/Research
+License :: BSD
```

Comment:
If possible, let's use Apache 2 for all the code in Terra

Reply:
yes everything should be apache 2.